src/backend/distributed/commands/multi_copy.c

```diff
@@ -1051,6 +1051,9 @@ RemoteFinalizedShardPlacementList(uint64 shardId)
 			shardPlacement->nodeName = nodeName;
 			shardPlacement->nodePort = nodePort;
 
+			/* we cannot know the nodeId, but it is not necessary at this point as well */
+			shardPlacement->nodeId = -1;
```

Comment:
We should probably just drop this feature (not in this PR).

Reply:
We've removed this logic: https://github.com/citusdata/citus/commit/90056f7d3cd9f2035dd90d38c658a7211151f3c9#diff-c532177d74c72d3f0e7cd10e448ab3c6L1123

So, I'll go ahead and convert `shardPlacement->nodeId` to uint